sdk/eventhub/event-hubs/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@azure/event-hubs",
-  "version": "2.1.1",
+  "version": "2.1.2",
```

Comment:
Is there a constants files somewhere we need to update?

Reply:
There is not! The version is grabbed from [package.json](https://github.com/Azure/azure-sdk-for-js/blob/7485069f5f701c16f8ea71f8ee4dd6e05cf82bd0/sdk/eventhub/event-hubs/src/util/constants.ts#L4)